README.md

```diff
@@ -56,7 +56,7 @@ route('events.venues.show', [event, venue]) // Returns '/events/1/venues/2'
 
 Practical AJAX example:
 
-```javascript
+```js
```

Comment:
Ah! ha, that's probably because I use javascript for everything and Daniel uses js for everything. Thanks!